### PR TITLE
Fix threshold configurator

### PR DESCRIPTION
**Pull request by evulins**
Fix styling and behaviour in Threshold configurator 
[Trello card with details ](https://trello.com/c/ZK3ltmBp/1859-calendar-page-thresholds-configurator-mobiledesktop)

### CODE REVIEW

File: app/javascript/react/components/ThresholdConfigurator/ThresholdConfigurator.style.tsx [hunk 1]
```diff
@@ -37,86 +39,52 @@ const RangeInput = styled.input<{
...
-  &:nth-child(6)::before,
-  &:nth-child(6)::after {
-    content: "";
-    position: absolute;
-    height: 24px;
-    border-radius: 5px;
-    margin-right: 10px;
+    width: 16px;
+    height: 16px;
+    background: #fff;
```
Comment (smialko) [inline app/javascript/react/components/ThresholdConfigurator/ThresholdConfigurator.style.tsx:69]: can you use` colors.white` for consistency?

**Review by smialko — APPROVED** [review]
Good job, just one minor comment from me :)

**Review by jankrzempek — CHANGES_REQUESTED** [review]
On the mobile - iPhone simulator it behaves weirdly. Sometimes it jump or flashes, and sometimes all the line turns red with no reason. Please see the following movie and screenshot.
Simulator Screenshot - iPhone 15 Pro - 2024-05-09 at 17 27 04

https://github.com/HabitatMap/AirCasting/assets/44276070/1b42f912-8716-41ae-a354-45505c20ac8d

**Review by smialko — APPROVED** [review]
Works well, looks great - very good work! :)

**Pull request merged**